miden-lib/asm/miden/contracts/faucets/basic_fungible.masm

```diff
@@ -72,9 +72,17 @@ export.distribute
     exec.faucet::mint
     # => [ASSET, tag, note_type, RECIPIENT, ...]
 
+    # store and drop the ASSET
+    mem_storew.3 dropw
```

Comment:
Similar comment as in the other file re `mem_storew` vs. `loc_storew`. Also, whey do we use `3` instead of `0`?

Reply:
I initially thought, `0` and `1` are already used. But this true for the faucet itself, but not for this context, I believe. I changed it now.